client/components/translatable/test/proptype.js

```diff
@@ -24,6 +24,22 @@ function assertFails( validator, { props }, propName = 'translatableString' ) {
 
 const Translatable = () => <span />;
 
+function testHOC( ComposedComponent ) {
+	const componentName = ComposedComponent.displayName || ComposedComponent.name || '';
+
+	return class extends React.Component {
+		static displayName = 'Localized(' + componentName + ')';
```

Comment:
A little brittle, not really a big deal as this is unlikely to change but it might be worth doing the real thing for clarity alone. (future developer: How do you get a localized component display name?)